Callout: Add background prop (#577)
steelthreads/agds-next: packages/callout/docs/overview.mdx

title: Callout
description: Callouts are an excerpt of text or an article that has been pulled out and used as a visual clue to draw the eye to the text. They are used to help direct a user's attention to important pieces of information.
group: Content

> storybookPath: /story/content-callout--on-light

Reviewer: ```suggestion
storybookPath: /story/content-callout--basic
```

	</Flex>
</Callout>
```

## Backgrounds

> The background of the Callout must pair with the background it is being placed on. If Callout is placed on a page with 'bodyAlt' background, please set `background="shadeAlt"`.

Reviewer: ```suggestion
The background of the `Callout` must match the background it is being placed on. For example, if `Callout` is placed on a 'bodyAlt' background, please set the `background` prop to `bodyAlt`.

```
